compose/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to install Docker Compose
+keywords: compose, orchestration, install, installation, docker, documentation
+title: Install Docker Compose
+toc_max: 2
+---
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
```

Comment:
I'd tweak this sentence to emphasise more that this page is about checking the installation scenario. It doesn't provide specific instructions on how to install compose. So maybe:
'You can run Compose on macOS, Windows and 64-bit Linux. Use the information on this page to check which installation scenario fits your needs.'

Reply:
You're right after the split into pages your comment makes sense. In your peer review feel free to always use the suggestion option.